packages/react-composites/tests/browser/chat/app/index.tsx

```diff
@@ -40,22 +53,27 @@ const uploadedFiles = params.uploadedFiles ? JSON.parse(params.uploadedFiles) :
 initializeFileTypeIcons();
 initializeIconsForUITests();
 
+let fakeChatAdapterModel = undefined;
+try {
+  fakeChatAdapterModel = JSON.parse(params.fakeChatAdapterModel);
+} catch (e) {
+  console.log('Query parameter fakeChatAdapterModel could not be parsed: ', params.fakeChatAdapterModel);
```

Comment:
willl console.logs mess up any clean logs we might want in CI?

should this be console.error?

Reply:
Removed this console log in the catch block and throwing an exception instead. Thanks.